src/main/java/bizbook/logic/commands/UndoCommand.java

```diff
@@ -0,0 +1,40 @@
+package bizbook.logic.commands;
+
+import bizbook.logic.commands.exceptions.CommandException;
+import bizbook.model.Model;
+
+/**
+ * Reverts the changes made to the address book.
+ */
+public class UndoCommand extends Command {
+
+    public static final String COMMAND_WORD = "undo";
+
+    public static final String MESSAGE_UNDO_SUCCESS = "Changes has been reverted!";
+    public static final String MESSAGE_UNDO_FAILURE = "Unable revert changes because "
+            + "there is no older version to revert to!";
```

Comment:
Could specify the max of 5 commands to be undone here

Reply:
I think is correct cause when we undo we are reducing the number of versions to 0